src/CONST.ts

```diff
@@ -3056,6 +3056,7 @@ const CONST = {
     // Character Limits
     FORM_CHARACTER_LIMIT: 50,
     STANDARD_LENGTH_LIMIT: 100,
+    STANDARD_LIST_ITEM_LIMIT: 8,
```

Comment:
When adding such const it should be good to also updated all the applicable usages to use it, maybe you can do it in another PR

Reply:
Yeah - today will add it with upcoming PR.